assets/styles.css

```diff
@@ -0,0 +1,325 @@
+  font-size: 20px;
+  line-height: 20px;
+  font-weight: bolder;
+}
+
+.footer-icon {
+  font-size: 26px;
+  color: #666;
+}
+
+.footer-lang {
+  color: #666;
+  font-size: 14px;
+  line-height: 15px;
+}
+
+.footer-link {
+  display: inline;
+  border-right: 1px solid #d9d9d9;
+  padding-right: 10px;
+  margin-right: 10px;
+  color: #666;
+}
+
+.headline {
+  color: #666;
+  color: rgba(3, 3, 3, 0.4);
+  font-family: Arial, Helvetica, sans-serif;
+  font: bold 15px/15px;
+  text-transform: uppercase;
+  padding: 15px 0;
+  border-bottom: 1px solid #d9d9d9;
+}
+
+.heading-text {
+  font-size: 18px;
+  line-height: 20px;
+  font-weight: 700;
+  padding: 0 9px;
+  border-right: 1px solid lightgray;
+}
+
+.logo {
+  font-family: "Abril Fatface", cursive;
+  font-size: 4rem;
+  font-weight: bold;
+  color: white;
+}
+
+.main-container {
+  background-color: white;
+}
+
+.navbar {
+  padding: 0px !important;
+  margin: 0 !important;
+}
+
+.navbar-buttons {
+  display: flex;
+  align-items: center;
+}
+
+.navbar-heading {
+  padding: 0 !important;
+}
+
+.navbar-date {
+  font-size: 16px;
+  color: #fff;
+  padding-top: 5vh;
+}
+
+.navbar-main {
+  background-color: #f72210 !important;
+  height: 95px;
+}
+
+.opinion-description {
+  color: #666;
+  font-size: 14px;
+  line-height: 21px;
+  text-transform: uppercase;
+  font-size: 13px/18px;
+  margin: 3px 0;
+}
+
+.opinion-title {
+  color: black;
+  font-weight: 800;
+  font-size: 12px;
+  line-height: 22px;
+}
+
+.placeholder-xs {
+  width: 100%;
+  height: 11vh;
+  background-color: gray;
+  float: left;
+  margin-right: 16px;
+  margin-bottom: 16px;
+  max-height: 75px;
+  max-width: 150px;
+}
+
+.placeholder-sm {
+  width: 100%;
+  height: 18vh;
+  background-color: gray;
+}
+
+.placeholder-md {
+  width: 100%;
+  height: 24vh;
+  background-color: gray;
+}
+
+.placeholder-pick {
+  width: 100%;
+  height: 80%;
+  background-color: gray;
+  min-height: 16vh;
+}
+
+.placeholder-round-md {
+  border-radius: 50%;
+  width: 55px;
+  height: 55px;
+  background-color: gray;
+  float: left;
+  margin: 8px;
+}
+
+.related-article {
+  border-top: 1px solid lightgray;
+  border-bottom: 1px solid lightgray;
+  padding: 8px 0;
+  font-size: 16px;
+  line-height: 20px;
+  color: #000;
+  text-align: center;
+  margin-bottom: 16px !important;
+}
+
+.row {
+  margin: 0 !important;
+}
+
+.section-headline {
+  max-width: 100% !important;
+  font: 19px/19px;
+  font-weight: 800;
+  border-bottom: 1px solid #d9d9d9;
+  padding: 8px 0px;
+}
+
+.section-icon {
+  padding: 2px 4px;
+  font-size: 1.66rem;
+  background-color: #f72210;
+  color: white;
+  float: left;
+  font-weight: bolder;
+  font: bold 19px/19px;
+  margin-right: 10px;
+}
+
+.section-pick {
+  height: 100%;
+}
+
+.section-picks {
+  border-top: 1px solid lightgray;
+  border-bottom: 1px solid lightgray;
+  padding: 16px 0;
+}
+
+.section-pick-category {
+  color: #f72210;
+  font-size: 20px;
+  line-height: 20px;
+  font-size: 16px/26px;
```

Comment:
Should be `font` :). Also remove lines 288 and 289 is you are using this property as they are not necessary.